Cargo.toml

```diff
@@ -23,25 +23,19 @@ exclude = ["doc", ".travis.yml"]
 [dependencies]
 atom = "0.3"
 fnv = "1.0"
-hibitset = "0.1"
+hibitset = { git = "https://github.com/slide-rs/hibitset" }
```

Comment:
You cannot publish crate with git dependencies anyways: All dependecies have to be from crates.io.